src/pages/IssueCollection/components/IssueCollectionForm.tsx

```diff
@@ -0,0 +1,156 @@
+import classNames from 'classnames';
+import { Button, MxLink } from 'components';
+import { CollectionTypeByNftEnum, DataTestIdsEnum } from 'localConstants';
+import { routeNames } from 'routes';
+import { NftEnumType } from 'types';
+import { useIssueCollectionForm } from '../hooks';
+import { IssueCollectionFieldsEnum } from '../types';
+
+export const IssueCollectionForm = () => {
+  const formik = useIssueCollectionForm();
+  const nft =
+    CollectionTypeByNftEnum[NftEnumType.NonFungibleESDT].toUpperCase();
+
+  const sft =
+    CollectionTypeByNftEnum[NftEnumType.SemiFungibleESDT].toUpperCase();
+
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      noValidate
+      className='d-flex flex-column'
+    >
+      <div className='flex flex-col gap-4 h-full'>
+        <div className='flex flex-col'>
+          <label
+            htmlFor={IssueCollectionFieldsEnum.tokenType}
+            className='block text-sm font-bold mb-2'
+          >
+            Type:
+          </label>
+          <div className='flex flex-row gap-4'>
+            <div>
+              <input
+                checked={
+                  formik.values[IssueCollectionFieldsEnum.tokenType] === nft
+                }
+                className='mr-2'
+                data-testid={DataTestIdsEnum.nftTypeInput}
+                defaultChecked
+                id={nft}
+                name={IssueCollectionFieldsEnum.tokenType}
+                onChange={formik.handleChange}
+                type='radio'
+                value={nft}
+              />
+              <label htmlFor={nft} className='text-sm'>
```

Comment:
Why is there another `label` element next to the `input` that has assigned a different `label` already?